lib/miq_automation_engine/service_models/miq_ae_service_container_route.rb

```diff
@@ -8,5 +8,9 @@ class MiqAeServiceContainerRoute < MiqAeServiceModelBase
     expose :labels,                   :association => true
     expose :is_tagged_with?
     expose :tags
+    expose :spec
```

Comment:
@kevensen 
Can the spec, spec=,annotations, tidy_provider_definition be moved into a mixin and then we can include it in 3 files